backend/compact-connect/lambdas/cloudfront-csp/test/config/index.js

```diff
@@ -0,0 +1,59 @@
+//
+//  index.js
+//  CompactConnect
+//
+//  Created by InspiringApps on 7/22/2024.
+//
+
+const chaiMatchPattern = require('chai-match-pattern');
+const chai = require('chai').use(chaiMatchPattern);
+const chalk = require('chalk');
+const lambdaLocal = require('lambda-local');
+const path = require('path');
```

Comment:
What's with the CommonJS `require` imports? Something to do with lambda requirements or environment?

Reply:
Basically just that CommonJS is still the default in Node; and maybe also that many libraries support CommonJS. Things like ESM tree-shaking aren't ever in play in our backend code. So I guess just simplicity overall.

Reply:
Got it, thanks!